cmd/manager/main.go

```diff
@@ -44,30 +45,35 @@ func init() {
 
 func main() {
 	var (
-		metricsAddr            string
-		enableLeaderElection   bool
-		probeAddr              string
-		secureMetrics          bool
-		enableHTTP2            bool
-		macPrefixesFile        string
-		insecure               bool
-		managerNamespace       string
-		probeImage             string
-		probeOSImage           string
-		registryPort           int
-		registryProtocol       string
-		registryURL            string
-		registryResyncInterval time.Duration
-		webhookPort            int
-		enforceFirstBoot       bool
-		enforcePowerOff        bool
-		serverResyncInterval   time.Duration
-		powerPollingInterval   time.Duration
-		powerPollingTimeout    time.Duration
-		discoveryTimeout       time.Duration
+		metricsAddr             string
+		enableLeaderElection    bool
+		probeAddr               string
+		secureMetrics           bool
+		enableHTTP2             bool
+		macPrefixesFile         string
+		insecure                bool
+		managerNamespace        string
+		probeImage              string
+		probeOSImage            string
+		registryPort            int
+		registryProtocol        string
+		registryURL             string
+		registryResyncInterval  time.Duration
+		webhookPort             int
+		enforceFirstBoot        bool
+		enforcePowerOff         bool
+		serverResyncInterval    time.Duration
+		powerPollingInterval    time.Duration
+		powerPollingTimeout     time.Duration
+		resourcePollingInterval time.Duration
+		resourcePollingTimeout  time.Duration
+		discoveryTimeout        time.Duration
 	)
 
 	flag.DurationVar(&discoveryTimeout, "discovery-timeout", 30*time.Minute, "Timeout for discovery boot")
+	flag.DurationVar(&resourcePollingInterval, "resource-polling-interval", 5*time.Second,
```

Comment:
I would set a higher default for the polling interval. 10 or even 20s. We don't want to DDoS the Redfish endpoint.